### ISSUE TITLE
Accessibility Issue: Inline frames must have a unique, non-empty 'title' attribute (WCAG 2.1 technique H64)

### ISSUE THREAD

**Issue by acalmelor** [nextcloud/external#257]
### Steps to reproduce
1. Go to External Page (For ex: Wiki) within NextCloud
2. Use a screen reader and navigate within.

### Expected behaviour
Use the "title" attribute of the <iframe> element to provide a unique, non-empty title that describes its contents.

**Why is this important?**
An inline frame’s title provides a label for the frame that describes its content to the user. With this information, a user can determine which frame to enter and explore in detail.

**Who does this affect?**
People using a screen reader, including blind, low vision, and neurodivergent people

### Actual behaviour
The inline frame does not have a 'title' attribute

`<iframe style="height: 746px;" allowfullscreen="" src="https://wiki.test.de" id="ifm">`

### What to do

- Use the "title" attribute of the <iframe> element to provide a unique, non-empty title that describes its contents. For example:

`<iframe src="https://cloudstorage.html" id="adiframe" name="adiframe" title="Cloud Storage"><a href="storage-ad.html"> Cloud storage</a></iframe>`

- It would be nice to have an additional field for the title on external sites setting.
